apps/tests/aws-runtime/test/test-service.ts

```diff
@@ -548,7 +548,10 @@ export const createAndDestroyWorkflow = workflow(
   }
 );
 
-export const counter = entity<{ n: number }>("counter2", z.any());
+export const counter = entity("counter3", {
+  schema: z.object({ n: z.number(), id: z.string() }),
```

Comment:
should the schema always be an object? Do we need the `z.object()` wrapper?

Reply:
Yes. It has to be an object. Making it a map would make the type less re-usable and now allow other fields like description on the object wouldn't it? I guess it is how you did other places, I'll see what you did.